### PR TITLE
BaSP-TG-11: admin test

### PR DESCRIPTION
**Pull request by GinaSchP**

### CODE REVIEW

File: src/tests/admins.test.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    });
+  });
+});
+describe('DELETE /delete/:id', () => {
+  test('response should return a 204 satus', async () => {
+    const response = await request(app).delete(`/api/admins/${idAdmin}`).send();
+    expect(response.status).toBe(204);
+    expect(response.body.error).toBeFalsy();
+  });
+  test('response should return a 400 satus', async () => {
```
Comment (CorinaArreguez) [inline src/tests/admins.test.js:122]: Gi, you have to change the message to 'response should return a 404 status'

**Review by CorinaArreguez — COMMENTED** [review]
Gina, I left you a comment on the line 122. Everytihng else seems perfect!

**Review by CorinaArreguez — APPROVED** [review]
Perfect Gina!

**Review by Julian-Cano — APPROVED** [review]
Everything looks good, great work!

**Review by CristianAlcober — APPROVED** [review]
Wonderful work Gina!

**Review by AgustinCarthery — APPROVED** [review]

**Pull request merged**